Source/Core/DolphinWX/Debugger/BreakpointWindow.cpp

```diff
@@ -57,33 +49,62 @@ class CBreakPointBar : public DolphinAuiToolBar
     AddTool(ID_ADDMC, "+MBP", m_Bitmaps[Toolbar_Add_MC]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::OnAddMemoryCheck, parent, ID_ADDMC);
 
-    AddTool(ID_LOAD, _("Load"), m_Bitmaps[Toolbar_Delete]);
+    AddTool(ID_LOAD, _("Load"), m_Bitmaps[Toolbar_Load]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::Event_LoadAll, parent, ID_LOAD);
 
-    AddTool(ID_SAVE, _("Save"), m_Bitmaps[Toolbar_Delete]);
+    AddTool(ID_SAVE, _("Save"), m_Bitmaps[Toolbar_Save]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::Event_SaveAll, parent, ID_SAVE);
   }
 
+  void ReloadBitmaps()
+  {
+    Freeze();
+
+    InitialiseThemedBitmaps();
+    for (int i = 0; i < ID_NUM; ++i)
+      SetToolBitmap(i, m_Bitmaps[i]);
+
+    Thaw();
+  }
+
 private:
   enum
   {
-    Toolbar_Delete,
+    Toolbar_Delete = 0,
+    Toolbar_Clear,
     Toolbar_Add_BP,
     Toolbar_Add_MC,
+    Toolbar_Load,
+    Toolbar_Save,
     Num_Bitmaps
   };
 
   enum
   {
-    ID_DELETE = 2000,
+    ID_DELETE = 0,
```

Comment:
Are you sure this doesn't introduce bugs? `2000` was likely selected specifically as an event ID. It might not cause bugs in the debugger UI, but it can definitely cause bugs if other UI controls/dialogs are visible, if no checking was done before changing this.

Reply:
This one tbh, I found it really weird because the watch window has the same structure and no explicit number were assigned so my guess was that it was only exclusive to the toolbars.  If it is a problem, I could rever this one and adjust the for loop to set the bitmaps and use 3000 for the watch window.  It felt like arbirtrary numbers and I didn't really undertand why the watch window was not doing the same thing.